tools/BuildDrop.ps1

```diff
@@ -0,0 +1,240 @@
+$buffer = 0
+
+for ($idx = 0; $idx -lt $content.Length; $idx++)
+{
+    # Flag that we will be looking at the entries for DH_WindowsAzurePowerShellVSFeed next
+    if ($content[$idx] -like "*VSFeed*")
+    {
+        $VSFeedSeen = $True
+    }
+
+    # Flag that we will be looking at the entry for DH_WindowsAzurePowerShellGet next
+    if ($content[$idx] -like "*PowerShellGet*")
+    {
+        $PSGetSeen = $True
+    }
+
+    # Check if we are looking at the DiscoveryHints for DH_WindowsAzurePowerShellVSFeed
+    # and if we have reached the end of the entry so we can add the new msi Product Code
+    if ($VSFeedSeen -and $content[$idx] -like "*</or>*")
+    {
+        $newContent[$idx] =     "      <discoveryHint>"
+        $newContent[$idx + 1] = "        <msiProductCode>$ProductCode</msiProductCode>"
+        $newContent[$idx + 2] = "      </discoveryHint>"        
+
+        # Change the buffer size to include the three lines just added
+        $buffer = 3
+    }
+
+    # Check if we are looking at the entry for DH_WindowsAzurePowerShellGet
+    if ($PSGetSeen -and $content[$idx] -like "*msiProductCode*")
+    {
+        $content[$idx] = "   <msiProductCode>$ProductCode</msiProductCode>"
+    }
+
+    $newContent[$idx + $buffer] = $content[$idx]
+}
+
+# Replace the contents of the current file with the updated content
+$result = $newContent -join "`r`n"
+$tempFile = Get-Item "DH_AzurePS.xml"
+
+[System.IO.File]::WriteAllText($tempFile.FullName, $result)
+
+# ==================================================================================================
+# Update the WebProductList_AzurePS.xml file
+# ==================================================================================================
+
+# Get the text for WebProductList_AzurePS.xml
+$content = Get-Content "WebProductList_AzurePS.xml"
+
+$PSGetSeen = $false
+
+for ($idx = 0; $idx -lt $content.Length; $idx++)
+{
+    # Flag that we will be looking at the entry for WindowsAzurePowerShellGet next
+    if ($content[$idx] -contains "  <productId>WindowsAzurePowershellGet</productId>")
+    {
+        $PSGetSeen = $true
+    }
+
+    # If we are in the WindowsAzurePowerShellGet entry, replace the necessary lines
+    if ($PSGetSeen)
+    {
+        if ($content[$idx] -like "*<version>*")
+        {
+            $content[$idx] = "  <version>$PSVersion</version>"
+        }
+
+        if ($content[$idx] -like "*<published>*")
+        {
+            $content[$idx] = "  <published>$($ReleaseDate)T12:00:00Z</published>"
+        }
+
+        if ($content[$idx] -like "*<updated>*")
+        {
+            $content[$idx] = "  <updated>$($ReleaseDate)T12:00:00Z</updated>"
+        }
+
+        if ($content[$idx] -like "*<trackingURL>*")
+        {
+            $content[$idx] = "        <trackingURL>http://www.microsoft.com/web/handlers/webpi.ashx?command=incrementproddownloadcount&amp;prodid=WindowsAzurePowershell&amp;version=$PSVersion&amp;prodlang=en</trackingURL>"
+        }
+    }
+
```

Comment:
Should have a condition here which ends the $PSGetSeen, to make this more robust over changed orderings in the file